dali/python/nvidia/dali/pipeline.py

```diff
@@ -963,7 +963,7 @@ def _run_input_callbacks(self):
             return
 
         for group in self._input_callbacks:
-            group.call_and_feed(self, self._iter)
+            group.call_and_feed(self, self._batch_size, self._iter)
```

Comment:
You've mentioned in the morning the idea of inferring batch size from other externalsources if there are a few of them and some work in batch mode (falling back to pipeline batch size as a last resort when there are no external sources working in batch mode) - do you handle that/do we want such a feature?

Reply:
No, it's not a part of this PR.

Reply:
Ok!